### PR TITLE
Serialize out all code style options.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/14323
Fixes https://github.com/dotnet/roslyn/issues/14322

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @dotnet/roslyn-ide

**Comment by dpoeschl** [comment]
This is missing a bunch of options under http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/CodeStyle/CodeStyleOptions.cs,bfdff4f8bcba6a30. Is that intentional?

**Comment by CyrusNajmabadi** [comment]
They are serialized in: src\Workspaces\Core\Portable\Execution\AbstractOptionsSerializationService.cs

**Comment by dpoeschl** [comment]
Thanks. :+1:

File: src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs [hunk 1]
```diff
@@ -54,5 +55,21 @@ internal static class CSharpCodeStyleOptions
         public static readonly Option<CodeStyleOption<bool>> PreferExpressionBodiedAccessors = new Option<CodeStyleOption<bool>>(
             nameof(CodeStyleOptions), nameof(PreferExpressionBodiedAccessors), defaultValue: CodeStyleOptions.TrueWithNoneEnforcement,
             storageLocations: new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(PreferExpressionBodiedAccessors)}"));
+
+        public static IEnumerable<Option<CodeStyleOption<bool>>> GetCodeStyleOptions()
```
Comment (Pilchie) [inline src/Workspaces/CSharp/Portable/CodeStyle/CSharpCodeStyleOptions.cs]: Should there be a similar change for VB?

**Pull request merged**